src/editing_tools/CropTool.vala

```diff
@@ -889,159 +915,165 @@ public class EditingTools.CropTool : EditingTool {
         int center_y = (top + bottom) / 2;
 
         switch (in_manipulation) {
-        case BoxLocation.LEFT_SIDE:
-            left = x;
-            if (get_constraint_aspect_ratio () != ANY_ASPECT_RATIO) {
-                float new_height = ((float) (right - left)) / get_constraint_aspect_ratio ();
-                bottom = top + ((int) new_height);
-            }
-            break;
-
-        case BoxLocation.TOP_SIDE:
-            top = y;
-            if (get_constraint_aspect_ratio () != ANY_ASPECT_RATIO) {
-                float new_width = ((float) (bottom - top)) * get_constraint_aspect_ratio ();
-                right = left + ((int) new_width);
-            }
-            break;
-
-        case BoxLocation.RIGHT_SIDE:
-            right = x;
-            if (get_constraint_aspect_ratio () != ANY_ASPECT_RATIO) {
-                float new_height = ((float) (right - left)) / get_constraint_aspect_ratio ();
-                bottom = top + ((int) new_height);
-            }
-            break;
-
-        case BoxLocation.BOTTOM_SIDE:
-            bottom = y;
-            if (get_constraint_aspect_ratio () != ANY_ASPECT_RATIO) {
-                float new_width = ((float) (bottom - top)) * get_constraint_aspect_ratio ();
-                right = left + ((int) new_width);
-            }
-            break;
+            case BoxLocation.LEFT_SIDE:
+                left = x;
+                if (get_constraint_aspect_ratio () != ANY_ASPECT_RATIO) {
+                    float new_height = ((float) (right - left)) / get_constraint_aspect_ratio ();
```

Comment:
Can use var here:
```suggestion
                    var new_height = ((float) (right - left)) / get_constraint_aspect_ratio ();
```

Other examples exist but not commented here as not changed

Reply:
Okay. I will update it then😁